app/scripts/directives/deployImage.js

```diff
@@ -378,13 +382,17 @@ angular.module("openshiftConsole")
               };
               nameTakenPromise.then(setNameTaken, setNameTaken).then(showWarningsOrCreate, showWarningsOrCreate);
             }, function(e) {
-              NotificationsService.addNotification({
-                id: "deploy-image-create-project-error",
-                type: "error",
-                message: "An error occurred creating project",
-                details: getErrorDetails(e)
-              });
               $scope.disableInputs = false;
+              if (e.data.reason === 'AlreadyExists') {
+                $scope.projectNameTaken = true;
+              } else {
+                NotificationsService.addNotification({
+                  id: "deploy-image-create-project-error",
+                  type: "error",
+                  message: "An error occurred creating project",
```

Comment:
We make the message a complete sentence with punctuation when possible.

```
"An error occurred creating project."
```

Reply:
Fixed